src/client/explorer.rs

```diff
@@ -216,7 +217,7 @@ impl From<&ContractDeposit> for ExplorerContractDeposit {
             nonce: obj.nonce,
             amount: obj.amount.into(),
             fee: obj.fee.into(),
-            sig: obj.sig.as_ref().map(|_| "Signed".into()), // TODO: Convert to hex
+            sig: Some(obj.sig.as_ref().map(|_| <&str as Into<String>>::into("signed")).unwrap().as_bytes().to_hex()), // DONE: TODO: Convert to hex
```

Comment:
Thanks for the PR!
This returns hex repr of "signed" string, not the actual signature bytes. right?
btw, we already have a `hex` library included, maybe you can use that instead of `rustc-serialize` :d

Reply:
Ok, I am thinking I should read up on maps in rust and get back to this.

Reply:
Do we need that mapped "Signed"?